robot_skills/src/robot_skills/base.py

```diff
@@ -139,6 +139,9 @@ def computePathLength(self, path):
                 distance += math.sqrt( dx*dx + dy*dy)
         return distance
 
+    def reset(self):
```

Comment:
Dit is toch overbodig? Local planner is ook een robotpart.

Reply:
Yes, bij deze aangepast!